New Unity Project/Assets/Script/LinkHandler.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using TMPro;
+
+[RequireComponent(typeof(TextMeshProUGUI))]
+public class LinkHandler : MonoBehaviour, IPointerClickHandler
+{
+    private TextMeshProUGUI pTextMeshPro;
+    private Canvas pCanvas;
+    private Camera pCamera;
+
+    public bool isLinkHighlighted { get { return pCurrentLink != -1; } }
```

Comment:
kan ook als `public bool isLinkHighlighted => pCurrentLink != -1;`  is wat netter vind ik :p